.code-samples.meilisearch.yaml

```diff
@@ -468,3 +450,70 @@ tenant_token_guide_search_sdk_1: |-
   front_end_client = MeiliSearch::Client.new('http://127.0.0.1:7700', token)
 
   front_end_client.index('patient_medical_records').search('blood test')
+getting_started_update_rankingRules: |-
+  client.index('movies').update_ranking_rules([
+    'exactness',
+    'words',
+    'typo',
+    'proximity',
+    'attribute',
+    'sort',
+    'release_date:asc',
+    'rank:desc'
+  ])
+getting_started_update_searchableAttributes: |-
+  client.index('movies').update_searchable_attributes([
+    'title'
+  ])
+getting_started_update_stop_words: |-
+  client.index('movies').update_stop_words(['the'])
+getting_started_check_task_status_md: |-
+  ```rb
+  client.index('movies').task(0)
```

Comment:
the `_md` should be removed, and the backtilt as a consequence as well

Reply:
Oh finally, thanks! 👯